db/seeds.rb

```diff
@@ -715,13 +715,26 @@
   nick = Participant.create({ andrewid: 'nharper', phone_number: 1713435788, user: nick_user })
   Membership.create({ organization: dtd_org, participant: nick })
 
-  puts 'Old Stuff...'
- tool = Tool.create({ name: 'Hammer', barcode: 7, description: 'it\'s a hammer' })
+  # Uncomment the line below and comment the creation of tool types and tools (so there is no conflicts)
+  #   to get a bunch of tools seeded into the database
```

Comment:
Yes, normally when in the development environment only a few tools are created. You can uncomment that line and comment the liens below to get a bunch of tools seeded for the development environment to test with.